Code/Core/CoreTest/Tests/TestTimer.cpp

```diff
@@ -32,7 +32,7 @@ void TestTimer::Validate() const
     Timer t;
     t.Start();
     const int64_t before = t.GetNow();
-    Thread::Sleep( 10 ); // sleep for 100ms
+    Thread::Sleep( 100 ); // sleep for 100ms
```

Comment:
On Apple Silicon, a 10ms sleep was not enough and would trigger the `TEST_ASSERT` on line 44. Maybe the timer resolution on the DTK machines is larger than that.